.github/workflows/check-runtime.yaml

```diff
@@ -47,11 +47,13 @@ jobs:
       - name: Build Runtime test suite for Lightning simulator
         if: ${{ matrix.simulator == 'lightning' }}
         run: |
-            cmake -S runtime -B build -DCMAKE_BUILD_TYPE=Release \
+            cmake -S runtime -B build -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Debug is required for coverage checks I assume? :)

Reply:
Yep, `gcov` required the C++ package to build in the release mode